ncloud/resource_ncloud_block_storage.go

```diff
@@ -160,7 +172,16 @@ func resourceNcloudBlockStorageUpdate(d *schema.ResourceData, meta interface{})
 
 	if d.HasChange("server_instance_no") {
 		o, n := d.GetChange("server_instance_no")
+
+		// If server instance attached block storage, detach first
 		if len(o.(string)) > 0 {
+			if d.Get("stop_instance_before_detaching").(bool) {
+				log.Printf("[INFO] Start Instance %s after detaching block storage", *ncloud.String(o.(string)))
```

Comment:
```suggestion
				log.Printf("[INFO] Start Instance %s after detaching block storage", o.(string))
```
Pointer로 변환할 필요 없이 Value로 사용하면 될것 같습니다~

Reply:
수정햇습니다~